docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp.md

```diff
@@ -0,0 +1,65 @@
+---
+id: udp
+title: 'Envoy Use Case: UDP'
+sidebar_label: 'Use Case: UDP'
+---
+
+Envoy allows to accept UDP requests.
+
+The Mia-Platform Console helps to configure the Envoy API Gateway with the advanced section.
+
+### Create the udp listener
+
+```yaml
+- "@type": type.googleapis.com/envoy.config.listener.v3.Listener
+  name: udp
+  address:
+    socket_address:
+      protocol: UDP
+      address: 0.0.0.0
+      port_value: 33333
+  udp_listener_config:
+    downstream_socket_config:
+      max_rx_datagram_size: 9000
+  listener_filters: 
+    - name: envoy.filters.udp_listener.udp_proxy 
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.filters.udp.udp_proxy.v3.UdpProxyConfig
+        stat_prefix: udp_stats  
+        cluster: udp_cluster
+        idle_timeout: 60s    
+```
+
+This configuration creates a listener named `udp` that accepts and proxies UDP traffic. 
+The listener listen on the port 33333 for incoming UDP packets accepting from any address.
```

Comment:
I would change "The listener listen on the port" into "The listener monitors the port"